packages/css/src/components/form/form.template.ts

```diff
@@ -0,0 +1,21 @@
+import { Form } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+
+import { formGroupInputTemplate } from './form-groups/input/form-group-input.template';
+
+export function formTemplate({ mode, formGroups }: Form<TemplateResult>) {
+  return html`
+    <form class="form-${mode}">
```

Comment:
Hier kan `form-vertical` uitkomen, en die bestaat niet.

Reply:
Ik denk dat je met classMap wel wat moet kunnen. Kijk even goed hoe `classMap` in andere files wordt geimporteerd, het is belangrijk dat de imports van `lit-html` eindigen op `.js`.

```
<form class=${classMap({ 'form-horizontal': mode === 'horizontal' })>
```

Reply:
`<form class=${ifDefined(mode === 'horizontal' ? 'form-horizontal' : undefined)}>`